src/js/view/controls/controlbar.js

```diff
@@ -11,13 +11,17 @@ import { prependChild, setAttribute, toggleClass, openLink } from 'utils/dom';
 import { timeFormat } from 'utils/parser';
 import UI from 'utils/ui';
 import { genId, FEED_SHOWN_ID_LENGTH } from 'utils/random-id-generator';
+import { isLanguageRtl } from 'utils/language';
 
-function text(name, role) {
+function text(name, role, language) {
     const element = document.createElement('span');
     element.className = 'jw-text jw-reset ' + name;
     if (role) {
         setAttribute(element, 'role', role);
     }
+    if (language && isLanguageRtl(language)) {
+        element.style.direction = 'rtl';
```

Comment:
this can be achieved without an additional repaint. If HTML attribute `dir` is set to auto, and the language is an rtl the text will automatically be set to rtl.

https://www.w3.org/International/tutorials/bidi-xhtml/index